coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 87.77
+    "covered_percent": 87.83
```

Comment:
Let's add the coverage directory to the gitignore. This is a lot to sift through and isn't really relevant to the code being committed.